recipes/gem3-mapper/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "3.6.1" %}
+{% set sha256 = "195f158691fe85bacc90b9cf848f203aeba1c56332516261da3774fee3f7ddbf" %}
+
+package:
+  name: gem3-mapper
+  version: {{ version }}
+
+source:
+  url: https://github.com/smarco/gem3-mapper/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - bzip2
+    - zlib
+  run:
+    - bzip2
+    - zlib
+
+test:
+  commands:
+          #    - gem-mappability -h 2>&1 |grep -e "-I <index_prefix>.*(mandatory)"
```

Comment:
anything wrong with this test?

Reply:
the particular executable is not included in this version. I started with an older version, but changed along the way. I will remove it.